model/src/main/java/org/cloudfoundry/identity/uaa/provider/AbstractExternalOAuthIdentityProviderDefinition.java

```diff
@@ -53,6 +53,7 @@ public enum OAuthGroupMappingMode {
     private boolean performRpInitiatedLogout = true;
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private String authMethod;
+    private boolean cacheJwks = true;
```

Comment:
@peterhaochen47 . The name of the attribute can be different, if you want or have a proposal for another name

Reply:
This works